pkg/sfu/buffer/buffer.go

```diff
@@ -404,42 +408,40 @@ func (b *Buffer) SetRTT(rtt uint32) {
 }
 
 func (b *Buffer) calc(pkt []byte, arrivalTime time.Time) {
```

Comment:
this runs per packet?

Reply:
Yes, per packet in the upstream direction.